syntax = "proto3";

message ImageFile {
	bytes value = 1;
	string similarity = 2;
}

service SimilarImage {
	rpc FindSimilar(ImageFile) returns (ImageFile) {}
}